keanu-examples/bayesianMethodsForHackers/src/main/java/entry/EntryPoint.java

```diff
@@ -0,0 +1,59 @@
+package entry;
+
+import examples.ABTesting;
+import examples.ChallengerDisaster;
+import examples.CheatingStudents;
+import examples.TextMessaging;
+import py4j.GatewayServer;
+
+/**
```

Comment:
why is all of this commented out?

Reply:
The commented out code is actually python for connecting to the keanu examples for graphing purposes